autoload/smoothie.vim

```diff
@@ -49,6 +49,14 @@ endfunction
 " already at the bottom.  Return 1 if cannot move any lower.
 function s:step_down()
   if line('.') < line('$')
+    " NOTE: the three lines of code following this comment block
+    " have been implemented as a temporary workaround for a vim issue
```

Comment:
How do we know it's temporary? Is there any ticket on Vim side we could track?

Reply:
Well... I like to consider _issue_s as temporary.. :smile: 
To be honest.. I wrote "temporary" because we don't know whether it is intended behavior, but to me it doesn't seem like intended behavior.

BTW, I did open a ticket at neovim https://github.com/neovim/neovim/issues/13080

Reply:
Awesome! Could you please link this ticket in the comment note?

Reply:
Will do.